app/Http/Controllers/CreatePostingPeriod.php

```
@@ -0,0 +1,184 @@
+        return $postingPeriod;
+    }
+
+    /**
+     * Create Monthly Period for next month
+     */
+    private function createNextMonthPostingPeriod(PostingPeriod $postingPeriod, Carbon $nextMonth): ?Period
+    {
+        $startOfNextMonth = $nextMonth->copy()->startOfMonth();
+        $endOfNextMonth = $nextMonth->copy()->endOfMonth();
+        
+        if (Period::CheckIfStatusIsOpenMonthly($postingPeriod)) {
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Period' in method 'createNextMonthPostingPeriod'. (Clean Code Rules)

(StaticAccess)